partiql-tests-data/success/eval/query/select/select.ion

```diff
@@ -0,0 +1,62 @@
+envs::{
+    sensors: [
+        {readings: [{v: 1.3}, {v: 2}]},
+        {readings: [{v: 0.7}, {v: 0.8}, {v: 0.9}]}
+    ]
+}
+
+{
+    name: "SELECT path",
+    statement: "SELECT r.v FROM sensors AS s, s.readings AS r",
+    assert: {
+        result: EvaluationSuccess,
+        output: $bag::[
+            {
+                v: 1.3
+            },
+            {
+                v: 2
+            },
+            {
+                v: 0.7
+            },
+            {
+                v: 0.8
+            },
+            {
+                v: 0.9
+            }
+        ]
+    }
+}
+
+{
+    name: "SELECT path with inline environment",
+    statement: "SELECT r.v FROM sensors AS s, s.readings AS r",
+    env: {
+        sensors: [
+            {readings: [{v: 1.3}, {v: 2}]},
+            {readings: [{v: 0.7}, {v: 0.8}, {v: 0.9}]}
+        ]
```

Comment:
How about changing the values here to ensure we read from the local env?

Reply:
Sounds good. Changed to use different values.